### PR TITLE
Update `repo-age` and `clean-repo-sidebar` spacing

### PR DESCRIPTION
**Pull request by fregante**
- Closes https://github.com/refined-github/refined-github/issues/5179

## Test URLs

https://github.com/fregante/webext-tools
https://github.com/refined-github/refined-github
https://github.com/rust-lang/rust

## Native

<table>
<tr>
	<th>Native
	<th>Before
	<th>After
<tr  valign=top>
	<td>
Screen Shot 1
	<td>Screen Shot 2
	<td>Screen Shot
</table>

### CODE REVIEW

File: source/features/clean-repo-sidebar.tsx [hunk 1]
```diff
@@ -61,26 +53,34 @@ async function hideEmptyPackages(): Promise<void> {
...
+async function init(): Promise<void> {
+	document.body.classList.add('rgh-clean-repo-sidebar');
+
+	await Promise.all([
+		hideReadmeLink(),
+		cleanReleases(),
+		hideEmptyPackages(),
+		hideLanguageHeader(),
+		hideEmptyMeta(),
+	]);
```
Comment (fregante) [inline source/features/clean-repo-sidebar.tsx:83]: Awaiting them all ensures that errors are reported as part of this feature

**Comment by fregante** [comment]
I ended up leaving all the links there because I don't want to deal with requests to restore it.

**Comment by fregante** [comment]
In the future I'd perhaps collapse them onto a single line:

Screen Shot 3

File: source/features/repo-age.tsx [hunk 2]
```diff
@@ -108,14 +108,14 @@ async function init(): Promise<void> {
 	// About a day old or less ?
 	const age = Date.now() - birthday.getTime() < 10e7
 		? fresh[Math.floor(Math.random() * fresh.length)]
-		: `${value} ${unit} old`;
+		: <><strong>{value}</strong> {unit} old</>;
```
Comment (kidonng) [inline source/features/repo-age.tsx:111]: This 👍 I was going to mention this when I see GitHub's new items but forgot 😅

**Comment by kidonng** [comment]
> In the future I'd perhaps collapse them onto a single line:
> 
> Screen Shot 3

I love the style but the eye icon may be mistaken for "views" instead of "watchers". I know it won't make sense for most people familiar with GitHub yet it could be ambiguous.

File: source/features/repo-age.tsx (hunk 2 above)
Comment (fregante) [reply source/features/repo-age.tsx:111]: "1 year" what though?

Like the eye icon doesn't stand on its own, neither does "year"

**Pull request merged**